doctrinal-works-v2/original/03-chapter-3.adoc

```diff
@@ -1046,8 +1046,15 @@ And you that despise him, despise your salvation,
 and lie under the judgment of the prophets and the Son, as you may see in the scriptures;
 and therefore, it is time for you to repent, and turn to him.
 
+[.postscript]
```

Comment:
I didn't know if this should be done a different way.

Reply:
yeah, i think this is fine 👍